indent/verilog_systemverilog.vim

```diff
@@ -45,7 +45,7 @@ let s:vlog_property       = '\(\(assert\|assume\|cover\)\s\+\)\@<!\<property\>'
 let s:vlog_sequence       = '\<sequence\>'
 let s:vlog_preproc        = '^\s*`ifn\?def\>'
 
-let s:vlog_case           = '\<case[zx]\?\>\s*('
+let s:vlog_case           = '\<\(case[zx]\?\>\s*(\|randcase\>\)'
```

Comment:
I think that what you want here is:

```vim
let s:vlog_case           = '\<\(rand\)\?case[zx]\?\>\s*('
```

Reply:
No, randcase has no "(selector)"
Syntax is:
``` verilog
randcase
1: x=4
1: x=5
endcase
```
I will take a look at the make file

Reply:
Should have checked the syntax first... :)